src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc

```diff
@@ -413,6 +494,20 @@ absl::StatusOr<bool> PrivateKeyAndCertificateMatch(
   return result;
 }
 
+absl::Status IsKeyCertPairListValid(const PemKeyCertPairList& pair_list) {
+  if (pair_list.empty()) {
```

Comment:
Should this be an error? We could have an empty pair list right?

Reply:
Makes sense. We can return an Ok status if it's empty. I'll document this "quirk" in the comments above its declaration so anybody calling it will take note.

Reply:
Sounds good. Thank you